jetty-jaas/src/main/java/org/eclipse/jetty/jaas/spi/LdapLoginModule.java

```diff
@@ -531,11 +538,6 @@ public boolean bindingLogin(String username, Object password) throws LoginExcept
 
     private SearchResult findUser(String username) throws NamingException, LoginException
```

Comment:
as findUser is private method, feel free to change its signature instead of introducing the findLdapUser method.